tests/pl/fan_HassTurnOn.yaml

```diff
@@ -5,6 +5,7 @@ tests:
       - "Uruchom w salonie wszystkie wentylatory"
       - "Włącz wentylatory w salonie"
       - "Uruchom wentylator w salonie"
+      - "Uruchom całe wentylatory w salonie"
```

Comment:
Same as roller shutters.
One may use "całe" with lights, but not with fans.
But on the other side, this is just a test case, so we can support this, even if someone won't use that case.

Reply:
As I mentioned earlier, this is just a test case, maybe someone speaks like this.
Let's leave it :)